pkg/constants/constants.go

```diff
@@ -0,0 +1,5 @@
+package constants
+
+const FirebaseUrl = "https://erouska-key-server-dev.firebaseio.com/"
```

Comment:
tyhle veci bych urcite delal konfigurovatelny pres ENV promenny, zatim to tam nech, ale pripadne pak udelam PR a predelam to :)